content/renderer/render_frame_impl.cc

```diff
@@ -4541,7 +4556,7 @@ void RenderFrameImpl::InitializeUserMediaClient() {
 WebMediaPlayer* RenderFrameImpl::CreateWebMediaPlayerForMediaStream(
     const blink::WebURL& url,
     WebMediaPlayerClient* client) {
-#if defined(ENABLE_WEBRTC)
+#if defined(ENABLE_WEBRTC) || defined(DISABLE_WEB_VIDEO)
```

Comment:
Just make sure the coding level right. Yes, if the Video disabled then WebRTC must doesn't work. 
Just make sure it can pass compiling if someone switch the 'DISABLE_WEB_VIDEO' and 'ENABLE_WEBRTC' independently to try build.